### DISCUSSION TITLE
Does Notify backup the trashed items too?

### DISCUSSION THREAD

**Discussion by serrq** [aritra-tech/Notify#187] [General]
Unfortunately for some reasons the restore method doesn't run in my device.

I wonder if the backup also include the deleted notes.
I mean, the ones that haven’t expired yet.

**Comment by aritra-tech** [comment]
> Unfortunately for some reasons the restore method doesn't run in my device.
> 
> I wonder if the backup also include the deleted notes. I mean, the ones that haven’t expired yet.

Actually, there is some bug in the backup & restore feature, can't get time to fix them.
I don't thing that backup also includes the deleted notes, will add this feature in the upcoming releases.
Thank you

**Reply by serrq** [reply]
There is a problem. A future problem.

If I backup all my notes, including the delete ones, they last for 28 days.

What does it happen if I try to restore a 40 days old backup? I will haven't the time to save my deleted (expired) notes because how the recover action end they expired, suddenly.

I propose too add an other day when one recover a backup and its deleted notes overcome the expiration date.

Is it a trivial problem?
What do you think about it?

**Reply by serrq** [reply]
So when one makes a recover, Notify updates the remaining days to "one day" to all those notes that have already expired. Sounds good?
